backend/src/core/config/settings.ts

```
@@ -11,15 +11,30 @@ const envVarsSchema = Joi.object()
       .required(),
     SERVER_PORT: Joi.number().default(8000),
     JWT_SECRET_KEY: Joi.string().required().description("JWT Secret Key"),
-    BCRYPT_SALT_ROUNDS: Joi.number()
+    JWT_ISSUER: Joi.string().default("aurora"),
+    JWT_AUDIENCE: Joi.string().default("aurora-users"),
+    JWT_ALGORITHM: Joi.string()
+      .valid("HS256", "HS384", "HS512", "RS256", "RS384", "RS512", "ES256", "ES384", "PS256", "PS384", "PS512")
+      .default("HS256"),
+    JWT_PRIVATE_KEY: Joi.string().when('JWT_ALGORITHM', {
+      is: Joi.string().pattern(/^(RS|ES|PS)/),
+      then: Joi.required().description("JWT private key (PEM) for asymmetric algorithms"),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)